app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -62,6 +62,11 @@ angular.module('Bastion.components').factory('Nutupane',
             params.page = $location.search().page || 1;
             params['per_page'] = $location.search()['per_page'] || entriesPerPage;
 
+            //Additional param to override initial load
+            self.overrideAutoLoad = params.overrideAutoLoad || false;
```

Comment:
> ( I am not familiar with the masterOnly property although I did see it in a few controllers)

Basically just setting it as a property on nutupane directily via either `nutupane.overrideAutoLoad = true;` or `nutupane.setOverrideAutoLoad(true);`.  I think I prefer the later.